### DISCUSSION TITLE
different enviroments make different result with same code

### DISCUSSION THREAD

**Discussion by estuday** [coqui-ai/TTS#4070] [General Q&A]
hi,
i have pull docker and pip install coqui-tts in local, the same code run in the docker and local make different results. 
code follows:
```python
import torch
from TTS.api import TTS
# Get device
device = "cuda" if torch.cuda.is_available() else "cpu"
# Init TTS
tts = TTS(model_path="./XTTS-v2",config_path="./XTTS-v2/config.json",progress_bar=True).to(device)
tts.tts_to_file(text="抄袭是没有未来的，遥遥领先", language="zh-cn", speaker_wav='./yu.wav', file_path="./output.wav")
```
the result is that in docker, the synthesis effect is acceptable, but in local, the synthesis effect is terrible.
i wonder why?

**Comment by eginhard** [comment]
There is no difference, it will just be random variation. If you fix the seed you should get the same outputs.
